internals/secrethub/clear_clipboard.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 // defaultClearClipboardAfter defines the default TTL for data written to the clipboard.
-const defaultClearClipboardAfter = 45 * time.Second
+const defaultClearClipboardAfter = 3 * time.Minute
```

Comment:
Is this also necessary for `read`?

Reply:
We discussed this in person: increasing the timeout for a might reduce the chance a little bit that the clipboard is cleared before using it, but at the same time, it also increases the chance of an accidental paste in the wrong location. Since using `--clip` for a read is a conscious decision, the fact that the clipboard is cleared is printed and it can easily be repeated by re-running the command, there is no real need to drastically increase this timeout (or at least not this much). 

---

For initializing services this is another story because this is not 100% repeatable; if the key gets cleared from the clipboard, it can never be retrieved and the service account must be recreated. So we could increase the timeout here, but it might be better to solve the underlying issue itself. Because the question is whether it is solved when we increase the timeout to 3 minutes.